Global.props

```diff
@@ -96,7 +96,7 @@
         <NugetVersionFilePath>$(MSBuildThisFileDirectory).nugetVersion</NugetVersionFilePath>
         <BuildNugetVersion Condition="Exists($(NugetVersionFilePath))">$([System.IO.File]::ReadAllText($(NugetVersionFilePath)))</BuildNugetVersion>
 
-        <CoreSdkVersion>2.2.0-beta2-build31642</CoreSdkVersion>
+        <CoreSdkVersion>2.2.0-beta3-build34090</CoreSdkVersion>
```

Comment:
Why are there dependencies on builds with build numbers?
For dependencies I think we should produce builds that are "public facing" without a build number in the name.